mock/src/tests.rs

```diff
@@ -259,6 +259,8 @@ fn send_to_disconnecting_node() {
     let mut a = Agent::node();
     let mut b = Agent::node();
 
+    println!("A={}, B={}", a.addr(), b.addr());
```

Comment:
is that suppose to be a println?

Reply:
Oops, must have left it there by accident. Will remove.